docs/control_center_docs/servo_module_getting_started.rst

```diff
@@ -0,0 +1,111 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _servo_module_start_guide:
+
+####################################################################
+Getting Started with Vertiq's Servo Firmware with IQ Control Center
+####################################################################
+
+Before completing the following getting started guide, please ensure that you have read and completed our :ref:`IQ Control Center 
+guide <control_center_start_guide>`. It walks you through Control Center installation, module configuration, and basic testing options available 
+to all Vertiq modules and firmware types. The following document is meant to provide additional configurations and testing options available only on Vertiq's 
+servo firmware.
+
+.. note::
+    The following images are captured using IQ Control Center version 1.5.2 as connected to a Vertiq 40-06 Gen 2
+
+.. warning::
+    Before setting any parameters, ensure that the module is held in place securely. If the module is unsecured it could move unexpectedly when 
+    it starts to spin. This could damage the module, its connectors, or any nearby observers.
+
+****************************************************
+Angle Controls and Maximum Angular Velocity
+****************************************************
+
+Vertiq's servo firmware provides several methods for commanding your module to specific positions. The most basic method is a 
+direct angle command. Angle commands tell the module to spin to a specific position at the velocity specified by the *Maximum Angular Velocity*. 
+To learn more about angle controls, see :ref:`control_displacements`.
+
+The Control Center's angle control testing parameter is available through the Testing tab, and the *Maximum Angular Velocity* parameter in the General tab.
+
+To test angle control with the Control Center:
+
+1.  Connect your module to the Control Center
+2.  Navigate to the General tab, and find the *Maximum Angular Velocity* parameter
+
+.. image:: ../_static/control_center_pics/servo_getting_started/servo_max_angular_velo.png
+
+3. Set *Maximum Angular Velocity* to 3.14 rad/s. For more information about the *Maximum Angular Velocity* parameter, see :ref:`control_displacements`.
```

Comment:
May be good to link straight to the section on maximum velocity here: https://iqmotion.readthedocs.io/en/feature-add_getting_started_manuals/manual/manual_angle_control_mechanisms.html#maximum-angular-speed